### ISSUE TITLE
[Questions] Refresh or enter the website for the first time, causing an error in the request

### ISSUE THREAD

**Issue by CharleeWa** [nuxt/movies#10]
The first time you enter or refresh a website, you will get this error. Then click on any of the other links, and the page will normally display.

**Command  line**

[Vue warn]: Unhandled error during execution of setup function at <AutoQuery key="tvpopular" query= { type: 'tv', title: 'Popular TV Shows', query: 'popular' } >

[nitro] [dev] [unhandledRejection] FetchError: request to https://movies-proxy.vercel.app/tmdb/tv/popular?page=1 failed, reason: connect ETIMEDOUT 31.13.75.12:443 ()

![截屏2022-10-20 17 42 53](https://user-images.githubusercontent.com/22477554/196916659-e211c2e7-27ca-4d0c-adfd-9445c7865b9d.png)

**Comment by antfu** [comment]
It seems you are trying this locally - are you on the latest commit? If so, can you share more info about this? I can't reproduce it on my side. Thanks

**Comment by CharleeWa** [comment]
@antfu Thank you. I used another win computer and the problem is still there,  it seems to be a common problem. I really don't have any more thoughts on this matter 🙈. 

The wrong steps:

- Start the app (pnpm dev)
- Two minutes later, the screen will display an error
- If I switch the menu, the page displays normally
- If I refresh, I have to wait 2 minutes for network requests
- Otherwise, I used a vpn proxy.
